irepela/homework_11/mine_sweeper.py

```diff
@@ -8,6 +8,30 @@
 }
 
 
+class GameException(Exception):
+    def __init__(self, data):
+        self.message = f"Invalid input data: {data}"
+
+    def __str__(self):
+        return self.message
+
+
+class OutOfFieldException(GameException):
+    def __init__(self, data, width, height):
```

Comment:
add typing and use more clear names

what is data?
what does width or height state for?
I do not see the code where this exception is used, so I need clear understanding what paremetears actually mean


coord, field        will be more clear